build.ts

```diff
@@ -67,8 +68,27 @@ try {
 
 	const isDebugBuild = argv.includes('--debug');
 	const buildType = isDebugBuild ? 'development' : 'production';
+	let platform: string = 'null' ;
```

Comment:
`platform` and `extension` should not be initiated as strings of 'null', use `undefined` or `null` but not `(string) 'null'`

Nit: Keep to code-style, no spaces before semi-colons.